src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs

```diff
@@ -141,6 +141,8 @@ private static bool GetLinqExpressionsBuiltWithIsInterpretingOnly()
 
         public static bool IsMetadataUpdateSupported => !IsNativeAot;
 
+        public static bool IsBoxedByRefLikeValuesSupported => !IsNativeAot;
```

Comment:
Subject/verb mismatch

```suggestion
        public static bool AreBoxedByRefLikeValuesSupported => !IsNativeAot;
```